src/System.Xml.ReaderWriter/src/System/Xml/Core/XmlTextReaderImplAsync.cs

```diff
@@ -934,26 +934,6 @@ private Task InitTextReaderInputAsync(string baseUriStr, Uri baseUri, TextReader
             return ReadDataAsync();
         }
 
-        private Task ProcessDtdFromParserContextAsync(XmlParserContext context)
```

Comment:
Call sites for this are now calling ProcessDtdFromParserContext, which is not the same as what this code was doing, e.g. it contains an extra case for case DtdProcessing.Parse.